### ISSUE TITLE
Support for station IDs unsupported by WAQI API

### ISSUE THREAD

**Issue by pawkakol1** [pawkakol1/worlds-air-quality-index#10]
[Part of issue#2 moved here](https://github.com/pawkakol1/worlds-air-quality-index/issues/2)

Support to integrate station using ID, what is greater than 13837 (Max station ID supported by WAQI API) using scrape

**Comment by alicialiali** [comment]
Same here: https://aqicn.org/station/@248629#/w/pt/

**Comment by pawkakol1** [comment]
@LeandroIssa, @grossqx, @joostvkempen
WAQI API supports station IDs grater than 13837 now, but not all station IDs.
I noticed, that they added API for stations from:
- CanAir.IO,
- stations from Citizen Science project luftdaten.info
These sources next to internal WAQI stations provide the biggest part of all available stations (I noticed, that some local sources provide by governments still are not supported by WAQI API)
Additional stations usually needs to put 'A' char as a prefix of ID station number during adding integration.

Unfortunately updated WAQI API supports new stations ID using requests by ID only, querying using geo localization still supports internal WAQI stations only.
